### PR TITLE
Revert an unneeded project file change.

### PR DESCRIPTION
**Pull request by StephenLPeters**
Revert an unneeded project file change introduced by #4464

### CODE REVIEW

File: dev/dll/Microsoft.UI.Xaml.vcxproj.filters [hunk 1]
```diff
@@ -147,225 +147,6 @@
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
```
Comment (ranjeshj) [inline dev/dll/Microsoft.UI.Xaml.vcxproj.filters:149]: how many of these do we need ?

Comment (StephenLPeters) [reply dev/dll/Microsoft.UI.Xaml.vcxproj.filters:149]: I'm not sure.. these are just the ones added my that previous PR, I just reverted the file to the state before that check in

**Review by ranjeshj — APPROVED** [review]

File: dev/dll/Microsoft.UI.Xaml.vcxproj.filters (hunk 1 above)
Comment (ranjeshj) [reply dev/dll/Microsoft.UI.Xaml.vcxproj.filters:149]: @llongley fixed this a while back, it might have regressed.

**Pull request merged**